.travis.yml

```diff
@@ -5,7 +5,9 @@ install:
   - pip install virtualenv
   - pip install tox
 script:
+  - pip3 install -e .
```

Comment:
It is - `tox` installs dependencies for itself but `python3` doesn't have access to those. Without this line, `python3 -m unittest` fails due to dependency issues.